repos/spack_repo/builtin/packages/gaudi/package.py

```diff
@@ -57,6 +57,19 @@ class Gaudi(CMakePackage, CudaPackage):
 
     maintainers("drbenmorgan", "vvolkl", "jmcarcell")
 
+    _cxxstd_values = (
+        conditional("14", when="@:39"),
+        conditional("17", when="@:39"),
+        conditional("20", when="@39:"),
```

Comment:
```suggestion
        conditional("20", when="@38:"),
```
per https://gitlab.cern.ch/gaudi/Gaudi/-/releases/v38r0